Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs

```diff
@@ -41,7 +41,7 @@ public class ShipControl : NetworkBehaviour
 
     float m_Acceleration = 12f;
 
-    float m_BulletLifetime = 2;
+    float m_BulletLifetime = 3;
```

Comment:
Why was this changed?

Reply:
This was changed because the inserting of the coroutine made the bullet disappear after the amount of seconds equal to the lifetime and 2 seconds were too short (it disappeared to early)

Reply:
Ok, so that relationship should be explained in a comment, and `m_BulletLifetime` should be a static readonly field equal to `2 + k_BulletDisappearTime` (where `k_BulletDisappearTime` is a constant that defines how much it takes for the bullet to disappear.

Without this info in the codebase, only you know about this relationship.